### PR TITLE
FMD-275: Drop GeospatialDim data_blob

### PR DESCRIPTION
**Pull request by MarcUsher**
### Change description
Drop the JSONB `data_blob` column from the GeospatialDim table, as the new `itl1_region_name` column now replaces the only field that was present in the `data_blob`.

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

**Review by samuelhwilliams — APPROVED** [review]
I think I'm fine with this in this case, but technically could raise errors (probably just in find) during rollout as discussed.

File: db/migrations/versions/035_drop_geospatial_data_blob.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        batch_op.drop_column("data_blob")
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("geospatial_dim", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), autoincrement=False, nullable=True)
```
Comment (wjrm500) [inline db/migrations/versions/035_drop_geospatial_data_blob.py:32]: Not sure it matters but original migration (no. 22) defined this column without `autoincrement` argument: `sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), nullable=True)`

**Review by wjrm500 — COMMENTED** [review]
Not got much to add other than this!

File: db/migrations/versions/035_drop_geospatial_data_blob.py (hunk 1 above)
Comment (MarcUsher) [reply db/migrations/versions/035_drop_geospatial_data_blob.py:32]: Hmmm good catch, that bit was auto-generated by the alembic migration! I'll try removing it and see if it thinks there are unapplied changes/anything breaks

**Pull request merged**